Dockerfile

```diff
@@ -12,12 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+FROM golang:1.12-buster AS build-env
+ENV GO111MODULE=on
+RUN apt-get update && apt-get install git --yes
+WORKDIR /build
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+RUN CGO_ENABLED=0 go build
+
 FROM alpine:3.9
-MAINTAINER Timothy St. Clair "tstclair@heptio.com"  
+MAINTAINER Timothy St. Clair "tstclair@heptio.com"
```

Comment:
Questionable :)

Reply:
Leaving in so we can issue PRs upstream.